### ISSUE TITLE
Disable caching very large files when uploading

### ISSUE THREAD

**Issue by wallart1** [astrada/google-drive-ocamlfuse#763]
I have a very large file (912 GB) that I want to upload to Google Drive. As it turns out, I don't have that much space in any of my file systems to hold the cache file. Is there a way that I can have google-drive-ocamlfuse disable the cache for files like this? Perhaps make the source file read-only during the upload rather than caching?

Thanks for this wonderful project.

EDIT: In case you are wondering, this file is a VeraCrypt volume. I am using VeraCrypt to encrypt my Google Drive.

**Comment by astrada** [comment]
No, sorry, it isn't possible because the Drive API doesn't allow to upload portions of a file. In your case, I think you should use a tool like https://rclone.org/

**Comment by askoriy** [comment]
@wallart1 You may use [EncFS](https://wiki.archlinux.org/title/EncFS) instead of VeraCrypt.
EncFS encrypts every file separately (and crypt filenames by default too), which makes it the best option to operate with google drive or any other cloud storage.
Encrypted EncFS tree could be easily synced as a regular folder to a mounted google drive.
Moreover, mounting EncFS directly on top of mounted google drive works fine for me :sunglasses:
